meta-lhg-wpe/recipes-wpe/wpeframework/wpeframework-plugins_git.bbappend

```diff
@@ -0,0 +1,2 @@
+# Enable CDMi
+PACKAGECONFIG_append = " opencdmi opencdmi_pr"
```

Comment:
Does this break our WPE 2016 solution which is based on Linaro OpenCDM? I expect that these components (opencdmi and opencdmi_pr) are not available in our WPE 2016 environment.

Reply:
No @ajutras-linaro, it won't break as WPE 2016 solution doesn't use this plugins recipe.